src/sat/boolean_problem.cc

```diff
@@ -13,8 +13,9 @@
 
 #include "sat/boolean_problem.h"
 
-#include "base/hash.h"
+#include <cmath>
```

Comment:
It is in there, see the line below. For some reason, the git diff considered to be better if it replaces "base/hash.h" with <cmath> and re-inserts a new line with #include "base/hash.h". :-(